software/database/src/test/java/org/apache/brooklyn/entity/database/mysql/MySqlClusterIntegrationTest.java

```diff
@@ -40,4 +42,22 @@ public void test_localhost() throws Exception {
             }
         }
     }
+
+    @Test(groups = {"Integration"})
+    public void testMasterInit() throws Exception {
+        try {
+            MySqlClusterTestHelper.testMasterInit(app, getLocation());
+        } finally {
+            for (Entity member : Iterables.getOnlyElement(app.getChildren()).getChildren()) {
+                String runDir = member.getAttribute(MySqlNode.RUN_DIR);
+                if (runDir != null) {
+                    Os.deleteRecursively(runDir);
+                }
+            }
+        }
+    }
+
+    private Location getLocation() {
+        return mgmt.getLocationRegistry().resolve("vagrant", ImmutableMap.of("minRam", "2G"));
```

Comment:
Needs comment to say what config is required in brooklyn.properties for this to work. Presumably it won't work for other people, or for jenkins?